internal/upload/koji/koji.go

```diff
@@ -445,7 +445,7 @@ func GSSAPICredentialsFromEnv() (*GSSAPICredentials, error) {
 func CreateKojiTransport(relaxTimeout uint) http.RoundTripper {
 	// Koji for some reason needs TLS renegotiation enabled.
 	// Clone the default http rt and enable renegotiation.
-	rt := &rh.RoundTripper{Client: rh.NewClient()}
```

Comment:
Is this not also used from `osbuild-koji`? You mentioned only `init` and `finalise` in the commit message, but it applies to everything, right?
Could you explain in the commit message exactly what the behaviour difference is? Is it only about logging, or does it improve the retry logic in some additional cases too?

Reply:
I got this now, thanks!